src/main/java/racingcar01/domain/Cars.java

```diff
@@ -0,0 +1,58 @@
+package racingcar01.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private static final String COMMA = ",";
+
+    private final List<Car> cars;
+
+    private Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static Cars of(String names) {
+        List<String> nameList = stringToList(names);
+        return new Cars(createCars(nameList));
+    }
+
+    public Cars moved() {
+        List<Car> movedCars = cars.stream()
+                                .map(car -> car.movedCar(CarMove.instance()))
+                                .collect(Collectors.toList());
+
+        return new Cars(movedCars);
+    }
+
+    public void forEach(Consumer<? super Car> cars) {
+        this.cars.forEach(cars);
+    }
+
+    public List<Car> winners() {
+        int maxDistance = maxDistance();
+
+        return cars.stream()
+                .filter(car -> car.distance() == maxDistance)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        return cars.stream()
                .filter(car -> car.isMaxDistance(maxDistance))
                .collect(Collectors.toList());
```
Car의 거리를 가져와서 비교하지말고
car 에게 최대거리와 같은지 물어보자.